src/entities/observation-entity.ts

```diff
@@ -164,10 +164,17 @@ export class Observation {
   })
   public accuracyOfDate: AccuracyOfDate;
 
-  // Related fields in access 'Lat deg', 'Lat min', 'Lat sec', 'Lon deg', 'Lon min', 'Lon sec',
-  @Length(15, 15, { message: equalLength(15) })
+  // Related fields in access 'Lat deg', 'Lat min', 'Lat sec'
+  @IsOptional()
+  @IsNumberString()
+  @Column('varchar', { nullable: true, default: null })
+  public latitude: string | null;
```

Comment:
sure that storing them as numbers will be better option

Reply:
Fixed.